src/app/components/SubmitBar/index.tsx

```diff
@@ -1,16 +1,21 @@
+import { levels } from '../../../config/questLevels';
```

Comment:
You're sure this needs to be in config?
So apparently some state object that a component depends upon should be outside of ```app/```. Really?

Reply:
If we fetch the level details from config, it would be easier to update in case we need to add more levels during the event